_episodes/03-create.md

```diff
@@ -215,15 +214,15 @@ draft.txt
 > 3.  When might you want to create a file this way?
 >
 > > ## Solution
-> > 1.  The touch command generates a new file called 'my_file.txt' in
+> > 1.  The `touch` command generates a new file called `my_file.txt` in
 > >     your home directory.  If you are in your home directory, you
```

Comment:
Again, the reference to the home directory is misleading. Whichever directory you ran the example in, the `ls` command will show the newly created file.